docs/semgrep-supply-chain/overview.md

```diff
@@ -50,27 +50,28 @@ _Figure 2_. Relationship between a Supply Chain rule, lockfile, CVE record, and
 
 ### Types of Semgrep Supply Chain findings
 
-If Semgrep Supply Chain determines that the codebase leverages a package
-containing a vulnerability, it generates a **finding**. If Semgrep Supply Chain
-also determines that there's a code pattern in the codebase that matches the
-vulnerability definition, then the finding is flagged as **reachable**
-(otherwise, it is **unreachable**).
-
-If Semgrep Supply Chain detects the use of a vulnerable version of a dependency,
-but it doesn't have a reachability rule, then the finding is **undetermined**
-(this is most common with vulnerabilities discovered prior to May 2022 or vulnerabilities with lower
-severity levels).
-
-Undetermined findings are generated by Semgrep Supply Chain's
-**[lockfile-only](/semgrep-supply-chain/glossary/#lockfile-only-rules)** rules,
-which are rules that do not perform reachability analysis. These findings are
-generated by checking the dependency's version listed in the lockfile or
-manifest against a list of versions with known vulnerabilities. These rules are
-on par with [GitHub Dependabot](https://github.com/dependabot).
-
-Specific dependency and code match findings are called **usages**, which are
-grouped by vulnerability. For each vulnerability, SCP also displays a CVE number
-corresponding to the [CVE program record](https://www.cve.org/About/Overview).
+Semgrep Supply Chains generates a **finding** any time it determines that your
+codebase leverages a package containing a vulnerability. In addition, Semgrep
```

Comment:
```suggestion
codebase uses or imports a package containing a vulnerability. In addition, Semgrep
```
I only noticed it now, but "leverages" is a little vague. The most common term in languages is `import` so I used that.